vars/Debian-10.yml

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: MIT
```

Comment:
would it make sense to name the file just "Debian", or are there relevant changes between Debian versions, or expected in the future?

Reply:
There are differences in python version in package name "python3" or just "python" (similar to Centos versions).

Reply:
Wait, if there is no Debian.yml, where is `__certificate_default_directory: /etc/ssl` going to be defined for other Debians?

Reply:
> There are differences in python version in package name "python3" or just "python" (similar to Centos versions).

Which version of Debian has `python3` in the names of the required packages? Debian 10 is the latest as of now. It it Debian testing that transitions to `python3` naming? Or were those the older versions that used `python3` and the newer ones transitioned to just `python`?

Reply:
On debian buster (10) both packages are available: 
https://packages.debian.org/buster/python-cryptography
https://packages.debian.org/buster/python3-cryptography

Running ansible on a cloud image on Debian 10, by default is using python2 and if I install python3-cryptography the execution just fails.

I haven't tested in any other Debian versions because Debian is actually out-of-scope for what we need to finish by June 15th. Do you think it's reasonable that we open an issue to add support to other Debian/Ubuntu versions and take this discussion there?

Reply:
Sure, you don't need to block it on this issue. i am just pointing out that if there are no known differences of packages required between Debian versions, then your comment
> There are differences in python version in package name "python3" or just "python" (similar to Centos versions).

is not accurate.

Reply:
I see. By having `Debian.yml` gives a chance that it might work in other Debian versions (even if not tested) while having `Debian-10.yml` enforces that it will work only on version 10.

I've renamed the file and will open an issue for testing that.

Reply:
It does not even enforce it, because on non-10 Debian it will pick `default.yml` and do the wrong thing without warning.

Reply:
(you probably meant a different thing by "enforces".)